vars/createSender.groovy

```diff
@@ -0,0 +1,30 @@
+import org.dotnet.helix.*
+def call(Map senderMap) {
+    return call(senderMap.CorrelationId, senderMap.WorkItemId, senderMap.CredentialsIdentifier)
+}
+def call(String credentialsIdentifier) {
+    return call(null, null, credentialsIdentifier)
+}
+def call(String correlationId, String credentialsIdentifier) {
+    return call(correlationId, null, credentialsIdentifier)
+}
+
+/* Create a helix event sender which is used to send events to the endpoint specified in the
+ * credentialsIdentifier
+ */
+def call(String correlationId, String workItemId, String credentialsIdentifier) {
+    withCredentials([string(credentialsId: credentialsIdentifier, variable: 'HelixEndpoint')]) {
```

Comment:
The credentialsId here appears unused.  Is this intentional or is it used in some other odd way?

Reply:
the credential is used on the next line 

`HelixEvent.Initialize(env.HelixEndpoint) `

Reply:
Oh, I'm an idiot...i mistakenly read that as another credential binding